### PR TITLE
Corrected typo in Spatial distance squareform

### PR DESCRIPTION
**Pull request by manipalsingh013**
Corrected formula in Spatial distance squareform from
v[{n choose 2}-{n-i choose 2} + (j-u-1)] 
to
v[{n choose 2}-{n-i choose 2} + (j-i-1)]

### CODE REVIEW

File: scipy/optimize/_minimize.py [hunk 1]
```diff
@@ -487,9 +487,9 @@ def minimize_scalar(fun, bracket=None, bounds=None, args=(),
     method : str or callable, optional
         Type of solver.  Should be one of
 
-            - 'Brent'     :ref:`(see here) <optimize.minimize_scalar-brent>`
-            - 'Bounded'   :ref:`(see here) <optimize.minimize_scalar-bounded>`
-            - 'Golden'    :ref:`(see here) <optimize.minimize_scalar-golden>`
+            - 'Brent'     :ref:`(see here) http://docs.scipy.org/doc/scipy/reference/optimize.minimize_scalar-brent.html#optimize-minimize-scalar-brent `
+            - 'Bounded'   :ref:`(see here) http://docs.scipy.org/doc/scipy/reference/optimize.minimize_scalar-bounded.html#optimize-minimize-scalar-bounded `
+            - 'Golden'    :ref:`(see here) http://docs.scipy.org/doc/scipy/reference/optimize.minimize_scalar-golden.html#optimize-minimize-scalar-golden `
```
Comment (larsmans) [inline scipy/optimize/_minimize.py]: ?

Comment (rgommers) [inline scipy/optimize/_minimize.py]: This is an incorrect fix, all links should be relative and not point to where we currently host the docs.

File: scipy/optimize/optimize.py [hunk 2]
```diff
@@ -1864,7 +1864,7 @@ def get_result(self, full_output=False):
             return self.xmin
 
 
-def brent(func, args=(), brack=None, tol=1.48e-8, full_output=0, maxiter=500):
+def brent(func, args=(), bracket=None, tol=1.48e-8, full_output=0, maxiter=500):
```
Comment (rgommers) [inline scipy/optimize/optimize.py]: This changes the public API, so will break code for people that use the keyword. Please remove.

Comment (rgommers) [inline scipy/optimize/optimize.py]: And the same applies to the other functions with this change.

**Comment by rgommers** [comment]
@manipalsingh013 can you remove the wrong commit from this PR? Use an interactive rebase, and then force-push to the same branch on your fork that you made this PR from.

**Comment by rgommers** [comment]
Note that gh-4634 tries to address the same issue. It actually has the exact same change. Given that and that this PR contains another unwanted commit, I'll close this one.

**Comment by rgommers** [comment]
@manipalsingh013 thanks for the PR. Picking an issue labeled ``easy-fix`` is good to get started, but preferably pick one that someone else hasn't submitted a fix for yet:)

**Pull request closed without merging**